src/SideNavigation/SideNav.js

```diff
@@ -57,7 +59,12 @@ SideNav.propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
     icons: PropTypes.bool,
-    selectedId: PropTypes.string
+    selectedId: PropTypes.string,
+    onItemSelect: PropTypes.func
+};
+
+SideNav.defaultProps = {
+    onItemSelect: () => { }
```

Comment:
Nice job adding the default empty function. 👍 This prop needs to be added to the `SideNav.propDescriptions` too.